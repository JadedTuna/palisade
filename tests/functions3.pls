in {
	high secret: int;
}
out {
	low x: int;
	low y: int;
}

fn foo(a: int) int { return a; }
fn bar(b: int) int { return foo(b); }

x = bar(10);
y = bar(secret);
