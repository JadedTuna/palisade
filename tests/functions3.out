[1;34mnote: [0mlabel of [1;34my[0m set to [1;93mhigh[0m
  13 | y [1;34m=[0m bar(secret);
       [1;34m~~[0m[1;34m^[0m
[[1;32m OK [0m] [1;93mlow [0m [1;34mx[0m is [1;93mlow[0m
[[1;31mFAIL[0m] [1;93mlow [0m [1;34my[0m is [1;93mhigh[0m
